packages/logger/src/index.ts

```diff
@@ -86,16 +44,23 @@ export const logger: ILogger = {
 };
 
 // tslint:disable-next-line: variable-name
-export const NSlogger = (namespace: string = ''): ILogger => ({
-  info: instance.info.bind(instance),
-  warn: instance.warn.bind(instance),
-  debug: LOG_LEVEL === 'debug' ? getDebugger(namespace) : () => {},
-  error: instance.error.bind(instance),
-});
+export const NSlogger = (namespace: string = ''): ILogger => {
+  instance.defaultMeta = {
+    ...instance.defaultMeta,
+    namespace,
+  };
```

Comment:
This is not great.. not sure what would be a better way to inject the namespace into the `defaultMeta`